src/shipchain/__tests__/testShipChainEncryptorContainer.ts

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2018 ShipChain, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+require('../../__tests__/testLoggingConfig');
+
+import 'mocha'
+
+import '../ShipChainEncryptorContainer'
```

Comment:
You are importing this twice.  Remove this line and keep the one below where we import the specific exported class from the module.